backend/core/src/units-summary/dto/units-summary.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { OmitType } from "@nestjs/swagger"
+import { UnitsSummary } from "../entities/units-summary.entity"
+
+export class UnitsSummaryDto extends OmitType(UnitsSummary, [] as const) {}
+export class UnitsSummaryCreateDto extends OmitType(UnitsSummaryDto, [] as const) {}
+export class UnitsSummaryUpdateDto extends OmitType(UnitsSummaryDto, [] as const) {}
```

Comment:
I think these can be fairly basic. If you take a look at `unit-types/dto/unit-type.dto.ts`, you can follow what was done there.

Reply:
The `UnitsSummaryDto` doesn't extend `AbstractEntity` like `UnitTypeDto` does, and none of the fields are generated. So I _think_ we may not need to omit any fields and the file is ok like this??

Reply:
@abbiefarr , looks like you are right. I think we're good then!